app/src/main/java/com/github/se/eventradar/ui/home/Home.kt

```diff
@@ -207,7 +225,7 @@ fun EventCard(event: Event) {
                       .weight(1f),
               verticalArrangement = Arrangement.Center) {
                 Text(
-                    text = event.name,
+                    text = event.eventName,
```

Comment:
nit : have "name" instead of "eventName" for the attribute to avoid the redondancy
(same goes for eventPhoto)

Reply:
I think eventName is better personally just to not get confused when used somewhere else